mbus-api/src/lib.rs

```diff
@@ -22,9 +22,8 @@ pub use mbus_nats::{
 pub use receive::*;
 pub use send::*;
 use serde::{de::StdError, Deserialize, Serialize};
-use smol::io;
 use snafu::{ResultExt, Snafu};
-use std::{fmt::Debug, marker::PhantomData, str::FromStr, time::Duration};
+use std::{fmt::Debug, io, marker::PhantomData, str::FromStr, time::Duration};
```

Comment:
Just out of my curiosity: why do we need std::io here now but didn't need it before ?
Sorry if I overlooked that somehow in your PR synopsis.

Reply:
Ah, I see: smol::io has been removed and replaced with std::io.